frontend/common/src/components/form/MultiSelect/MultiSelectField.test.tsx

```diff
@@ -104,6 +104,7 @@ describe("MultiSelectFieldV2", () => {
           { value: "BAZ", label: "Baz" },
           { value: "BAM", label: "Bam" },
         ]}
+        doNotSort
```

Comment:
I tried to get this to work but I have no clue how to test this component because the markup it generates is just non-sense lol. It is rendering `<div>`'s with no discernable role to select them with. Probably a more pressing concern for a11y as well 🤔 

I check [WAI Comobox Pattern](https://www.w3.org/WAI/ARIA/apg/patterns/combobox/examples/combobox-autocomplete-list/) and they clearly have a `<li>` with `role=option` which is what `<Combobox />` is doing.

Reply:
🤷